pandas/core/reshape/concat.py

```diff
@@ -200,7 +200,7 @@ def concat(
 
         .. versionchanged:: 1.0.0
 
-           Changed to not sort by default.
+            Changed to not sort by default.
```

Comment:
there's still this unrelated whitespace (and others below) - if you click "File changed" it'll show you everything you've changed, and you can just check your pull request is minimal and only changes what needs changing